syntax="proto3";    //版本号
package xmsg;       //命名空间

enum MsgType
{
    NONE_DO_NOT_USE = 0;
    MSG_LOGIN_REQ = 1;  //客户端用户名和密码登录请求 XLoginReq
    MSG_LOGIN_RES = 2;  //服务端响应回复消息
    
    MSG_MAX_TYPE = 65535; 
}